x/chainmain/client/cli/query.go

```diff
@@ -74,11 +74,18 @@ Example:
 				return allBalancesErr
 			}
 
-			baseUnit := coinParser.GetBaseUnit()
+			baseUnit, err := sdk.GetBaseDenom()
+			if err != nil {
+				return err
+			}
 			baseAmount := res.Balances.AmountOf(baseUnit)
+			humanCoin, err := sdk.ConvertDecCoin(sdk.NewDecCoin(baseUnit, baseAmount), config.HumanCoinUnit)
```

Comment:
comment about ConvertDecCoin would be good

Reply:
It converts the coin to the specified denomination, for example `"1000000basecro" -> "1cro"`